docs/api/synthetics/monitors/delete-monitor-api.asciidoc

```diff
@@ -37,7 +37,7 @@ DELETE /api/synthetics/monitors/monitor1-id
 
 ==== Bulk Delete Monitors
 
-You can delete multiple monitors by sending a list of config ids to a DELETE request to the `/api/synthetics/monitors` endpoint.
+You can delete multiple monitors by sending a list of config ids to a POST request to the `/api/synthetics/monitors/_bulk_delete` endpoint.
```

Comment:
Because these routes are publicly documented we will need to first deprecate the existing route and expose a the new `/_bulk_delete` route. And before we implement a deprecation we need to get it reviewed.

Reply:
My understanding is that now that we aren't deprecating the `api/synthetics/monitors` route, but just crating a new `_bulk_delete` route, we do not need to do this. Is that correct?